sass/MLFacet/_MLFacetValues.scss

```diff
@@ -0,0 +1,47 @@
+@import '../Variables';
+
+.coveo-ml-facet-values {
+  margin: 15px 0;
+  padding: 0;
+}
+
+.coveo-ml-facet-value {
+  list-style: none;
+  margin-bottom: 6px;
+
+  &.coveo-selected {
+    .coveo-checkbox-span-label,
+    .coveo-checkbox-span-label-suffix {
+      font-weight: bold;
+    }
+  }
+
+  .coveo-checkbox-label {
+    width: 100%;
+  }
+
+  .coveo-checkbox-span-label {
+    white-space: nowrap;
+    text-overflow: ellipsis;
+    overflow: hidden;
+  }
+
+  .coveo-checkbox-span-label-suffix {
+    margin-left: 8px;
+    color: $color-light-grey;
+  }
+
+  .coveo-checkbox-button:focus {
+    & ~ .coveo-checkbox-span-label,
+    & ~ .coveo-checkbox-span-label-suffix {
+      opacity: 0.5;
```

Comment:
Would it be better to store the opacity in a variable and reuse it on line 44?

Reply:
Yes